frontend/src/components/landing/Reviews.tsx

```diff
@@ -0,0 +1,254 @@
+  function handleTouchEnd() {
+    // user swiped left
+    if (touchStart - touchEnd > 150) {
+      scrollReview(currentReview, userReviews, "left");
+    }
+
+    // user swiped right
+    if (touchStart - touchEnd < -150) {
+      scrollReview(currentReview, userReviews, "right");
+    }
+  }
+
+  const userReviews: UserReview[] = [
+    {
+      name: "Jon",
+      rating: 5,
+      text: l10n.getString("landing-review-user-one-review"),
+    },
+    {
+      name: `${l10n.getString("landing-review-anonymous-user", {
+        user_id: "17064608",
+      })}`,
+      rating: 5,
+      text: l10n.getString("landing-review-user-two-review"),
+    },
+    {
+      name: `${l10n.getString("landing-review-anonymous-user", {
+        user_id: "16464118",
+      })}`,
+      rating: 5,
+      text: l10n.getString("landing-review-user-three-review"),
+    },
+    {
+      name: `${l10n.getString("landing-review-anonymous-user", {
+        user_id: "17361666",
+      })}`,
+      rating: 5,
+      text: [
+        l10n.getString("landing-review-user-four-review-list-1"),
+        l10n.getString("landing-review-user-four-review-list-2"),
+        l10n.getString("landing-review-user-four-review-list-3"),
+        l10n.getString("landing-review-user-four-review-list-4"),
+      ],
+    },
+  ];
+
+  const reviewCount = 328;
+  const { name, text, rating } = userReviews[currentReview];
+
+  const scrollReview = (
+    count: number,
+    reviews: UserReview[],
+    direction: Direction
+  ): void => {
+    if (direction === "right") {
+      setCurrentReview(count < reviews.length - 1 ? count + 1 : 0);
+    } else if (direction === "left") {
+      setCurrentReview(count > 0 ? count - 1 : reviews.length - 1);
+    }
+
+    setScrollAnimationDirection(direction);
+  };
+
+  // We create an array with a length of 5
+  // which represents the max number of stars a rating can achieve.
+  // We iterate that array and map to stars based on rating passed
+  // into function.
+  const renderStarRating = (rating: Rating) =>
+    [...Array(5)].map((star, index) =>
+      index < rating ? (
+        <StarIcon className={styles.star} key={index} alt="" />
+      ) : (
+        <StarIcon className={styles["empty-star"]} key={index} alt="" />
+      )
+    );
+
+  // Render a bulleted list if review is an array of strings.
+  const renderReview = (text: Review) => {
+    if (!Array.isArray(text)) return <p>{text}</p>;
+
+    return (
+      <ul>
+        {text.map((item, index) => (
+          <li key={index}>{item}</li>
+        ))}
+      </ul>
+    );
+  };
+
+  return (
+    <section id="reviews" className={styles.wrapper}>
+      <div className={styles.reviews}>
+        <div className={styles["left-container"]}>
+          <div className={styles["logo-container"]}>
+            <img className={styles["logo"]} src={FxBrowserLogo.src} alt="" />
+            <p className={styles["logo-title"]}>
+              {l10n.getString("-brand-name-firefox-browser")}
```

Comment:
@Vinnl do you think this `-brand-name-firefox-browser` isn't working because we are using `pendingTranslations.ftl` instead of `en/app.ftl`?

Reply:
I don't think so; technically, we load both, with the former just being a fallback for if a string isn't available in the latter. I expect that Fluent variables are just intentionally local to the translation file.